odoo/_monkeypatches/evented.py

```diff
@@ -0,0 +1,35 @@
+"""
+Running mode flags (gevent, prefork)
+
+This should be imported as early as possible.
+It will initialize the `odoo.evented` variable.
+"""
+import odoo
+import sys
+
+odoo.evented = False
+if len(sys.argv) > 1 and sys.argv[1] == 'gevent':
+    sys.argv.remove('gevent')
+    import gevent.monkey
+    import psycopg2
+    from gevent.socket import wait_read, wait_write
+    gevent.monkey.patch_all()
```

Comment:
This is not monkeypatching, this is part of odoo basic setup. even the following part is adding a callback, not monkeypatching.

Reply:
Ok, moved outside of monkeypatches.